ruff.toml

```diff
@@ -2,18 +2,37 @@ target-version = "py311"
 
 lint.select = ["ALL"]
 
+lint.preview = true
+
 lint.ignore = [
     "ANN101",   # Missing type annotation for `self` in method
+    "B909",     # Mutation to loop iterable `self._attributes` during iteration
```

Comment:
Can you add the rules that will be fixed later after a TODO comment, that way we'll be able to identify and work individually.

Reply:
I think that the best approach is to open a draft PR on the rule you are working on, so we avoid working on the same